agent/utils/compare_versions_test.go

```diff
@@ -185,6 +185,11 @@ func TestVersionMatches(t *testing.T) {
 			selector:       ">=1.5.0",
 			expectedOutput: true,
 		},
+		{
+			version:	"1.17",
```

Comment:
Shouldn't you move this to `engine/dockerclient` package as well?

Reply:
yeah, this test should not be here, will remove it. There is a general test added in `engine/dockerclient` for getting Docker versions from `minAPIVersion`.